networking/k8s_nmstate/k8s-nmstate-observing-node-network-state.adoc

```diff
@@ -11,4 +11,5 @@ Node network state is the network configuration for all nodes in the cluster.
 
 include::modules/virt-viewing-network-state-of-node.adoc[leveloffset=+1]
 
+[id="virt-viewing-network-state-of-node-console"]
```

Comment:
```suggestion
[id="virt-viewing-network-state-of-node-console_{context}"]
```

Reply:
@JoeAldinger  Actually, this file, k8s-nmstate-observing-node-network-state.adoc, is irrelevant - it's not in the distro, replaced by k8s-nmstate-updating-node-network-config.adoc. So I removed that change.